md_file_standards/q02_number-input/q02_number-input.md

```diff
@@ -38,7 +38,6 @@ server: |
 part1:
  type: number-input
  answer: part1
```

Comment:
@gbovett I don't get why this is is `answer: part1` this line is redundant right?

Reply:
resolved by removing in #20